kitsune/wiki/migrations/0010_change_locale_bn_BD_to_bn.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.22 on 2019-08-16 16:54
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def change_locale_bn_bd_to_bn_forwards(apps, schema_editor):
+    Document = apps.get_model('wiki', 'Document')
+    DraftRevision = apps.get_model('wiki', 'DraftRevision')
+    Locale = apps.get_model('wiki', 'Locale')
+
+    Document.objects.all().filter(locale='bn-BD').update(locale='bn')
```

Comment:
We need to do the same for all the articles in `bn-IN` that do not exist in `bn-BD`.

Reply:
Probably we need to also delete any documents/revisions/locales that may exist under the old locales